File: android/app/src/main/java/com/devnokiyo/actioncableapp/enums/RoomName.kt
package com.devnokiyo.actioncableapp.enums


enum class RoomName(val rawValue: String) {
    Tuna("tuna"),
    Egg("egg"),
    SalmonRoe("salmon_roe");

    companion object {
        var rooms: Array<String> = arrayOf()
            get() = arrayOf(RoomName.Tuna.rawValue, RoomName.Egg.rawValue, RoomName.SalmonRoe.rawValue)
    }
}

File: ios/ActionCableApp/Enums/RoomName.swift
enum RoomName: String {
    case tuna = "tuna"
    case egg = "egg"
    case salmonRoe = "salmon_roe"
    
    static var names: Array<String> {
        return [RoomName.tuna.rawValue, RoomName.egg.rawValue, RoomName.salmonRoe.rawValue]
    }
}

